### ISSUE TITLE
Change flag for UKR

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#997]
Here is the image he mentioned resized to 30x20.

![ukr](https://cloud.githubusercontent.com/assets/7934273/12860708/ce9623fe-cca1-11e5-9a1c-004c88a4b671.png)

Wall Post by https://tatoeba.org/eng/user/profile/thorn0

https://tatoeba.org/eng/wall/show_message/25424#message_25424

The Ukrainian flag used on Tatoeba is wrong. The upper stripe has to be darker. You can find the correct colors on Wikipedia or on the site of the Ukrainian parliament.

https://en.wikipedia.org/wiki/Ukraine
http://rada.gov.ua

Where should I report such issues?

For comparison, here is what we are currently using.
![ukr 1](https://cloud.githubusercontent.com/assets/7934273/12860938/2d0004b8-cca3-11e5-946e-cb987d1486b5.png)

**Comment by RyckRichards** [comment]
It's here. I will change that as soon as possible.

**Comment by trang** [comment]
To  be clear, the icon is not wrong, and this is not a bug. We have no rules that says that when we reuse a country flag, we must keep it exactly identical.

In the past, all the language icons had to go through some re-adjustments: the colors were intentionally made "lighter", so that the language icon doesn't come out too flashy.
Also a 1px grey border was added to the right and bottom sides, so that icons such as the Japanese language icon would have a clearer border.

That's why the Ukranian language icon looks ligther. Later on these rules were not necessarily followed anymore. That's why some other languages have a language icon that has the exact same colors than the original flag.

If we want to keep it consistent, we would need to review ALL the language icons. Which is something I'm planning to do, i.e. to replace someday all the existing language icon, with are PNG images, by SVG images. Any additional effects would be defined via CSS rather than editing the image itself. @RyckRichards has started to help out gathering SVG images for each language, but since it's rather low priority, it may take a while before we can migrate to SVG language icons.

I have no problem with replacing the current language icon for Ukranian if this is what Ukranian contributors prefer. But if this is not urgent, then we might as well wait until we re-harmonize all the language icons at once.

**Comment by ckjpn** [comment]
My guess is that it would have taken you less time to upload the replacement than to write your reply.
It's no big deal to me, but at least one member has requested this change.

**Comment by trang** [comment]
> My guess is that it would have taken you less time to upload the replacement than to write your reply.

Yes, but I would have posted the reply regardless of whether or not we replace the icon... My goal was to share information, not to find reasons for not replacing the icon :)

> It's no big deal to me, but at least one member has requested this change.

We can't always apply every small changes that our members request. Even if it takes 2 seconds. For instance last time we replaced the Spanish language icon, someone else got unhappy about the change. Just because one person wants something doesn't meant that it'll be the right thing to do.

**Comment by RyckRichards** [comment]
I have contacted native speakers and users that posted Ukranian sentences in order to know if they will agree or not.

**Comment by RyckRichards** [comment]
I have contacted some members and they either approve the new flag or don't really care about this change. Changing it right now.
